website/content/api-docs/peering.mdx

```diff
@@ -0,0 +1,322 @@
+---
+layout: api
+page_title: Cluster Peering - HTTP API
+description: The /peering endpoints allow for managing cluster peerings.
+---
+
+# Cluster Peering - HTTP API
+
+~> **Cluster peering is currently in technical preview:** Functionality associated
+with cluster peering is subject to change. You should never use the technical
+preview release in secure environments or production scenarios. Features in
+technical preview may have performance issues, scaling issues, and limited support.
+
+The functionality described here is available only in
+[Consul](https://www.hashicorp.com/products/consul/) version 1.13.0 and later.
+
+## Generate a Peering Token
+
+This endpoint generates a peering token.
+
+| Method  | Path                 | Produces           |
+| ------- | -------------------- | ------------------ |
+| `POST`  | `/peering/token`     | `application/json` |
+
+The table below shows this endpoint's support for
+[blocking queries](/api-docs/features/blocking),
+[consistency modes](/api-docs/features/consistency),
+[agent caching](/api-docs/features/caching), and
+[required ACLs](/api#authentication).
+
+| Blocking Queries | Consistency Modes | Agent Caching | ACL Required     |
+| ---------------- | ----------------- | ------------- | ---------------- |
+| `NO`             | `none`            | `none`        | `none`           |
+
+### JSON Request Body Schema
+
+- `PeerName` `(string: <required>)` - The name assigned to the peer cluster.
+  The `PeerName` is used to reference the peer cluster in service discovery queries
+  and configuration entries such as `service-intentions`. This field must be a
+  valid DNS hostname label.
+
+- `Partition` `(string: "")` - <EnterpriseAlert inline /> The admin partition that the
+  peering token is generated from. Uses `default` when not specified.
+
+- `Datacenter` `(string: "")` - Specifies the datacenter. Defaults to the
```

Comment:
```suggestion
- `Datacenter` `(string: "")` - Specifies the datacenter. Defaults to the
```
Getting a little lost - this is the datacenter that you want to create peer connection with, correct? Can we say "target datacenter" or "external datacenter"? "Specifies the target datacenter for the peer connection."?

Reply:
This is the datacenter where the token is generated. The datacenter that the `Partition` is in.